cagallo/rancid-tomatillos
# Feature/react router ect

## What Features are Being Added?
- Install and Implement React Router.
- Add `<BrowserRouter>` to index.js , `<Route>s` to App.js for both the landing page and movie view page, and converted all MoviePoster's to be contained in `<Link>s`.

## What is Being Refactored?
- The MovieView component is now a class based component once again rather than a functional component so we could take advantage of the componentDidMount lifecycle method.
- Removed all logic dealing with click events. 
- Removed conditional rendering that was previously responsible for changing page views. 

## Where should the reviewer start?
- Trace the router changes starting in index.js then going to App.js, MovieContainer.js, and MovieView.js respectively to see how all the changes come into play. 

## How should this be manually tested?
- Install React Router and load up the application and test to see whether the page still functions/the URL updates when navigating through the page. 

## What are the relevant tickets?
- User flows/stories for iteration 4

## Any Other Notes/Questions/Problems?
- We still need to implement the back button, and I think we should look at converting the MovieView render function into smaller sub components.
